### PR TITLE
CSS Two Column Layout - Michael Van Doorn

### PR DESCRIPTION
**Pull request by Michaelvxd**
CleanShot 2022-12-20 at 20 22 32

### CODE REVIEW

**Comment by Michaelvxd** [comment]
This works but I'm using 4 columns per row when the point was to use two? 🤔

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -13,35 +13,57 @@
   <div class="container">
     <div class="row">
       <div class="column-header">
-        <h1>Scale, Speed, Agility</h1>
+        <h1>Scale&comma; Speed&comma; Agility</h1>
         <p>We've dedicated thousands of man hours to making healthcare
           institutions in the Helium network run 10x more efficiently
-          and deliver and unparalleled quality of care
+          and deliver and unparalleled quality of care&period;
```
Comment (scrumptiousdev) [inline css-two-column-layout/index.html:19]: These are just fine but usually we do not have to worry about converting `.` / `,` / `"` / `'` / `-` into HTML entities. Those are very commonly used for text content. We should use entities for special characters that is not commonly used like trademarks or registered.

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
So for this challenge, we should also use rows and columns. So we can basically break this down into either 4 columns with flex wrap or 2 rows with 2 columns each.

**Review by scrumptiousdev — APPROVED** [review]
Approved!

**Pull request merged**